src/Microsoft.ML.TimeSeries/SsaAnomalyDetectionBase.cs

```diff
@@ -59,23 +59,23 @@ public static Double SquaredDifference(Double actual, Double predicted)
         {
             switch (errorFunction)
             {
-            case ErrorFunction.SignedDifference:
-                return SignedDifference;
+                case ErrorFunction.SignedDifference:
+                    return SignedDifference;
 
-            case ErrorFunction.AbsoluteDifference:
-                return AbsoluteDifference;
+                case ErrorFunction.AbsoluteDifference:
+                    return AbsoluteDifference;
 
-            case ErrorFunction.SignedProportion:
-                return SignedProportion;
+                case ErrorFunction.SignedProportion:
+                    return SignedProportion;
 
-            case ErrorFunction.AbsoluteProportion:
-                return AbsoluteProportion;
+                case ErrorFunction.AbsoluteProportion:
+                    return AbsoluteProportion;
 
-            case ErrorFunction.SquaredDifference:
-                return SquaredDifference;
+                case ErrorFunction.SquaredDifference:
+                    return SquaredDifference;
 
-            default:
-                throw Contracts.Except(ErrorFunctionHelpText);
```

Comment:
We should call this "the great case-switch statement indenter" PR. 😆

Reply:
I call it the carpal tunnel PR :P